joomla/joomla-cms
# [6.0] Notification email on admin registration approval

This is a rework for j4 of this PR #20282

### Summary of Changes
The user who are manually actived will receive a notification by email informing his account has been activated. 
There is also the option of being able to manually send a reminder email to the user even though the account is already activated (in case the user do not login in the page after X time)
The mail template used is `com_users.registration.user.admin_activated`

### Testing Instructions
Apply the PR and after run `npm run build:js -- build/media_source/com_users/js`

1. Open the frontend and create a new user account
2. Open the User edit view inside the administration, and you will notice a new button in the toolbar
3. Use the button and check the user was actived and also a email sent to the email of the new user

-  A email will be sent also if the user is actived in the Users list view using the **Actions -> Active** option

## Actual result BEFORE applying this Pull Request
You can active a user, but not notification is sent.

### Expected result AFTER applying this Pull Request
You can active a user, but a notification is sent.

#### Accept suggestions for:
- Button text
- Button icon
- Button color

![user](https://user-images.githubusercontent.com/3956472/212749081-d0a357c8-b194-449d-b0f8-e57db6f7430a.png)

### Link to documentations
Please select:
Page: [Users:_Edit_Profile](https://docs.joomla.org/Help4.x:Users:_Edit_Profile) -> Document the new toolbar button
Page: [Users](https://docs.joomla.org/Help4.x:Users) -> Here just need mention that active the user will send a email with the notification.
